### ISSUE TITLE
Unable to outline jsonld data

### ISSUE THREAD

**Issue by angelo-v** [SolidOS/solid-panes#254]
When requesting a JSON-LD document, the data browser only shows an error message:

```
Outline.expand: Unable to fetch <https://angelo.veltens.org/public/toots/hello-solid>: HTTP error: Status undefined (undefined)
```

Requested URI: https://angelo.veltens.org/public/toots/hello-solid

![image](https://user-images.githubusercontent.com/340725/98134588-32838900-1ebf-11eb-9b2c-b3d0b9bd29d4.png)

Note that, altough it says "HTTP error", there are only 2xx status codes in the browser network tab, no errors.

**Comment by angelo-v** [comment]
Might be a follow up error of https://github.com/linkeddata/rdflib.js/issues/408 and/or https://github.com/linkeddata/rdflib.js/issues/343

**Comment by jeff-zucker** [comment]
I'm pretty sure this is resolved by https://github.com/linkeddata/rdflib.js/pull/509 which is included in rdflib v2.2.7

**Comment by angelo-v** [comment]
Awesome! Finally SolidOS is able to render the toot I sent to mastodon!

https://angelo.veltens.org/public/toots/hello-solid#note

![image](https://user-images.githubusercontent.com/340725/126677426-f3fbb043-1a87-4a85-a9be-83ff8ad76f7e.png)
